languages/julia/exercises/concept/fibonacci-iterator/.docs/instructions.md

```diff
@@ -0,0 +1,86 @@
+In this exercise you're going to write an iterator that iterates through the first `n` elements of the [Fibonacci sequence](https://en.wikipedia.org/wiki/Fibonacci_number).
+
+## Fibonacci sequence
+
+The Fibonacci sequence is defined recursively.
+Each element of the sequence is the sum of the two previous elements.
+The first two elements of the sequence are set to 1.
+In maths notation:
+
+$$
+
+a_1 = a_2 = 1\\
+a_n = a_{n-1} + a_{n-2}
+
+$$
+
+For example, the third element would be $a_3 = a_1 + a_2 = 1 + 1 = 2$, the fourth would be $a_4 = a_2 + a_3 = 1 + 2 = 3$, and so on.
+
+The first elements of the sequence are $1,~1,~2,~3,~5,~8,~13,~21,~34,~55,~89,~\dots$
+
+!!! note
+    Some sources define the first elements as $a_0 = 0$ and $a_1 = 1$.
+    However in the context of this exercise, we define the sequence without a 0-th element.
```

Comment:
@iHiD do you think this is understandable to someone without a maths background?